docs/conf.py

```diff
@@ -53,8 +54,10 @@
 modindex_common_prefix = ["template_project."]
 
 # html theme options
-html_static_path = ["_static"]
+html_title = f"{project} {release}"
 html_logo = "_static/images/logo.png"
+html_theme = "qiskit-ecosystem"
+html_theme_options = {"sidebar_hide_name": True}
```

Comment:
This only is a good default if the project has a log that includes the project name in it. I'd recommend removing it.

Reply:
Done, I passed the option to False